### DISCUSSION TITLE
Cant seem to query data about every lap since a given start date

### DISCUSSION THREAD

**Discussion by VBrix** [br-g/openf1#105] [Q&A] [answered]
Hi, 
Cant seem to get the hang on the filtering. What seem to be wrong with how i try filtering the data in the URL? :)

response <- GET('https://api.openf1.org/v1/laps?date_start>2023-09-01')
parsed_data <- fromJSON(content(response, 'text'))
head(parsed_data)

**Comment by br-g** [comment, marked as answer]
Hello,
Your URL seems correct, but you're asking for too many results at once.
This this instead: https://api.openf1.org/v1/laps?date_start>2024-09-01

**Reply by VBrix** [reply]
Thanks for the help :)
